### PR TITLE
"Heading has non-empty accessible name" (ffd0e9): Add Accessibility support note on valid but empty aria-labelledby

### LINKED ISSUE #1203: Failed example 1 has accessible name [ffd0e9]
```
<span id="label"></span>
<h1 aria-labelledby="label">ACT rules</h1>
```
https://act-rules.github.io/rules/ffd0e9#failed-example-1

This example actually has accessibile name as the current node `<h1>` is an element with a Text node, return its textual contents: 
here's the G step from accessible name and description computation:
> Otherwise, if the current node is a Text node, return its textual contents.

On the other hand according to [Name and Description](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd) the determinations of content contributes to the names string can be from two possible values i.e. "author" or "contents". As name from "author" value is empty but it still have name from "contents" value.

### PR DESCRIPTION
**Pull request by Jym77**
An an accessibility support note on discrepancies when `aria-labelledby` refers to an existing element with empty content.

Closes issue(s):

- closes #1203

Need for Final Call:
This will not require a Final Call (change to accessibility support only).

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/heading-non-empty-accessible-name-ffd0e9.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ _There are currently no assumptions._
 
 - Implementation of [Presentational Roles Conflict Resolution][] varies from one browser or assistive technology to another. Depending on this, some elements can have a [semantic role][] of `heading` and fail this rule with some technology but users of other technologies would not experience any accessibility issue.
 
+- The [accessible name and description computation][] suggest that if an `aria-labelledby` attribute refers to an existing but empty element, the computation should stop and return and empty name without defaulting to the next steps, but isn't very clear about this. Several user agents and assistive technologies chose to use the next step in the computation in this case.
```
Comment (WilcoFiers) [inline _rules/heading-non-empty-accessible-name-ffd0e9.md:47]:
```suggestion
- The [accessible name and description computation][] suggest that if an `aria-labelledby` attribute refers to an existing but empty element, the computation should stop and return and empty name without defaulting to the next steps. Several user agents and assistive technologies chose to use the next step in the computation in this case.
```

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
Minor suggestion.

File: _rules/heading-non-empty-accessible-name-ffd0e9.md [hunk 2]
```diff
@@ -44,6 +44,8 @@ _There are currently no assumptions._
 
 - Implementation of [Presentational Roles Conflict Resolution][] varies from one browser or assistive technology to another. Depending on this, some elements can have a [semantic role][] of `heading` and fail this rule with some technology but users of other technologies would not experience any accessibility issue.
 
+- The [accessible name and description computation][] suggest that if an `aria-labelledby` attribute refers to an existing but empty element, the computation should stop and return and empty name without defaulting to the next steps. Several user agents and assistive technologies chose to use the next step in the computation in this case.
```
Comment (carlosapaduarte) [inline _rules/heading-non-empty-accessible-name-ffd0e9.md:47]:
```suggestion
- The [accessible name and description computation][] suggest that if an `aria-labelledby` attribute refers to an existing but empty element, the computation should stop and return an empty name without defaulting to the next steps. Several user agents and assistive technologies chose to use the next step in the computation in this case.
```

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
typo

**Review by WilcoFiers — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Review by ajanec01 — APPROVED** [review]

**Review by kasperisager — APPROVED** [review]

**Pull request merged**